### DISCUSSION TITLE
Small Font Size for R Code Chunks in Callout Boxes

### DISCUSSION THREAD

**Discussion by produnis** [quarto-dev/quarto-cli#11130] [Q&A] [answered]
### Description

I’m using Quarto to render my books in HTML format, and I’ve noticed that the font size of R code chunks inside callout boxes is smaller compared to the rest of the document. I would like to increase the font size for these code chunks.

Could you please advise what changes I need to make to my SCSS file to adjust the font size for R chunks inside callout boxes?

**Comment by mcanouil** [comment]
You need to use your favourite browser inspect/developer mode and hover on your target.
You'll get everything needed.

Additionally, see https://quarto.org/docs/output-formats/html-themes.html

**Reply by produnis** [reply]
I’m finding it challenging to figure this out myself because everything is so div-nested.

**Reply by produnis** [reply, marked as answer]
Ha, Chatgpt told me to add

```
pre code {
  font-size: 1em;
}

```
and this works for now

**Reply by mcanouil** [reply]
What's challenging?

Higher you go, wider is the impact.

https://github.com/user-attachments/assets/6fac4fd6-3baf-4dd8-9a63-282a08ee32a3

**Reply by mcanouil** [reply]
Note that if you want to do custom theme change in HTML, you need to learn (S)CSS, at least some basics: https://www.w3schools.com/Css/
(Shortcut via AI is only a temporary solution)
